src/scripts/ui/inline-roll-links.ts

```diff
@@ -315,7 +315,7 @@ export const InlineRollLinks = {
 
                 const actor = resolveActor(foundryDoc, link);
                 if (actor || pf2Traits) {
-                    const origin: Record<string, unknown> = {};
+                    const origin: Record<string, unknown> = (flags.pf2e.origin as Record<string, unknown>) ?? {};
```

Comment:
```suggestion
                    const origin: Record<string, unknown> = flags.pf2e.origin ?? {};
```

Reply:
This won't compile because flags.pf2e is record<string, unknown> so .origin is unknown type